src/http-api/index.js

```diff
@@ -23,6 +26,11 @@ function HttpApi (repo, config) {
   this.log = debug('jsipfs:http-api')
   this.log.error = debug('jsipfs:http-api:error')
 
+  // Setup debug metrics collection
+  const collectDefaultMetrics = prometheusClient.collectDefaultMetrics
+  collectDefaultMetrics({ timeout: 5000 })
+  prometheusGcStats(prometheusClient.register)()
```

Comment:
got it. Still, we should not add stuff that is not necessary by default. Can these be in a separate file that doesn't get loaded unless a env var is set?